use std::error::Error;

pub fn run() -> Result<String, Box<Error>> {
    Ok("json".to_string())
}
